### DISCUSSION TITLE
How to get these jpg which will change with time or how to make them?

### DISCUSSION THREAD

**Discussion by zilong-ding** [t1m0thyj/WinDynamicDesktop#403] [Q&A]
yes, i know how to simply make a ddw by adding jpg and writing a json. and i want to konw how to get these jpg or how to make them?

**Comment by t1m0thyj** [comment]
One option is to use the "Create theme from single image" button in the [DDW Theme Creator](https://ddw-theme-creator.vercel.app/) (thanks to @gdstewart for creating it).

You can also download HEIC wallpapers from a website like [dynamicwallpaper.club](https://dynamicwallpaper.club/) and convert them to a DDW file using the same tool.

Apart from finding dynamic themes that others have already created, if you want to create your own dynamic wallpapers from scratch you'd need to get into timelapse photography 🙂
